go.mod

```diff
@@ -1,68 +1,115 @@
-	github.com/opentracing/basictracer-go v1.0.0
+	github.com/olekukonko/tablewriter v0.0.4
+	github.com/opentracing-contrib/go-grpc v0.0.0-20191001143057-db30781987df // indirect
+	github.com/opentracing/basictracer-go v1.1.0
 	github.com/opentracing/opentracing-go v1.1.1-0.20200124165624-2876d2018785
 	github.com/pkg/errors v0.9.1
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/prometheus/alertmanager v0.20.0
-	github.com/prometheus/client_golang v1.5.1
+	github.com/prometheus/client_golang v1.6.0
 	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.9.1
+	github.com/prometheus/common v0.10.0
 	github.com/prometheus/prometheus v1.8.2-0.20200407102557-cd73b3d33e06
 	github.com/sercand/kuberesolver v2.4.0+incompatible // indirect
-	github.com/uber/jaeger-client-go v2.20.1+incompatible
+	github.com/shirou/gopsutil v2.20.4+incompatible // indirect
+	github.com/sirupsen/logrus v1.6.0 // indirect
+	github.com/uber/jaeger-client-go v2.23.1+incompatible
 	github.com/uber/jaeger-lib v2.2.0+incompatible
-	go.elastic.co/apm v1.5.0
-	go.elastic.co/apm/module/apmot v1.5.0
-	go.uber.org/automaxprocs v1.2.0
-	golang.org/x/crypto v0.0.0-20191206172530-e9b2fee46413
+	github.com/weaveworks/common v0.0.0-20200512154658-384f10054ec5 // indirect
+	go.elastic.co/apm v1.8.0
+	go.elastic.co/apm/module/apmot v1.8.0
+	go.elastic.co/fastjson v1.1.0 // indirect
+	go.mongodb.org/mongo-driver v1.3.3 // indirect
+	go.uber.org/atomic v1.6.0 // indirect
+	go.uber.org/automaxprocs v1.3.0
+	golang.org/x/crypto v0.0.0-20200510223506-06a226fb4e37
+	golang.org/x/mod v0.3.0 // indirect
+	golang.org/x/net v0.0.0-20200520004742-59133d7f0dd7 // indirect
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
-	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
+	golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a
+	golang.org/x/sys v0.0.0-20200519105757-fe76b779f299 // indirect
 	golang.org/x/text v0.3.2
-	golang.org/x/tools v0.0.0-20200306191617-51e69f71924f // indirect
-	google.golang.org/api v0.20.0
-	google.golang.org/genproto v0.0.0-20200305110556-506484158171
-	google.golang.org/grpc v1.27.1
+	golang.org/x/time v0.0.0-20200416051211-89c76fbcd5d1 // indirect
+	golang.org/x/tools v0.0.0-20200519205726-57a9e4404bf7 // indirect
+	google.golang.org/api v0.24.0
+	google.golang.org/genproto v0.0.0-20200519141106-08726f379972
+	google.golang.org/grpc v1.29.1
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
 	gopkg.in/fsnotify.v1 v1.4.7
-	gopkg.in/yaml.v2 v2.2.8
-	gopkg.in/yaml.v3 v3.0.0-20200121175148-a6ecf24a6d71
+	gopkg.in/ini.v1 v1.56.0 // indirect
+	gopkg.in/yaml.v2 v2.3.0
+	gopkg.in/yaml.v3 v3.0.0-20200506231410-2ff61e1afc86
+	honnef.co/go/tools v0.0.1-2020.1.4 // indirect
+	howett.net/plist v0.0.0-20200419221736-3b63eb3a43b5 // indirect
+	k8s.io/api v0.18.2 // indirect
+	k8s.io/apimachinery v0.18.2 // indirect
+	k8s.io/client-go v1.5.1 // indirect
+	k8s.io/utils v0.0.0-20200520001619-278ece378a50 // indirect
+	sigs.k8s.io/yaml v1.2.0 // indirect
```

Comment:
we need `go mod tidy`

Reply:
It already has been. AFAIU those `// indirect` are due to modules that have no `go.mod` files.